redditnav.css

```diff
@@ -1,15 +1,115 @@
-a.mfb-component__button--main, a.mfb-component__button--child {
-  border-radius: 50% !important;
+#redditNavContainer {
+  position: fixed;
+  bottom: 15px;
+  z-index: 1000000;
 }
 
-.ion-chevron-down:before {
-  color: white;
+#redditNavContainer.right {
+  right: 20px;
 }
 
-.ion-chevron-up:before {
-  color: white;
+#redditNavContainer.left {
+  left: 20px;
 }
 
-.mfb-component--hide {
+#redditNavContainer.hide {
   display: none;
 }
+
+#redditNavContainer > .button {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  padding: 10px 5px;
+}
+
+#redditNavContainer > .button.main {
+  z-index: 1;
+}
+
+#redditNavContainer > .button.main ~ .button {
+  transition: all 500ms 50ms cubic-bezier(0.68, -0.55, 0.265, 1.55);
+}
+
+#redditNavContainer:hover > .button.main ~ .button {
+  transform: translateY(-70px);
+}
+
+#redditNavContainer > .button > a {
+  position: relative;
+  display: block;
+  height: 56px;
+  width: 56px;
+  background-color: currentColor;
```

Comment:
Yeah it's a fun little thing I learned over the summer :)
if you're curious, take a look at this [video](https://www.youtube.com/watch?v=8xjR7QXQKJ0&feature=youtu.be&t=41m4s).  it is all about the CSS3 color spec.